Sources/BraintreePayPal/BTPayPalEditRequest.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// Options for the PayPal Edit FI flow
+/// - Warning: This feature is currently in beta and may change or be removed in future releases.
+public class BTPayPalEditRequest {
+    let token: String
+
+    /// Initializes a PayPal Edit Request for the Edit FI flow
+    /// - Parameters:
+    ///   - token: Required: Used to initiate tokenize call to edit funding instrument in customer's PayPal account.
```

Comment:
Do we have more details yet on how merchants get this token. It will be via the Server SDKS, but do we know the method name?

Reply:
I do see the spec for the  server side endpoint but I have questions on it, I posted the q on confluence page.
I'm good with TODO for now.